src/accelerate/utils/imports.py

```diff
@@ -175,6 +175,10 @@ def is_datasets_available():
     return _is_package_available("datasets")
 
 
+def is_peft_available():
```

Comment:
We need to add this to `utils.__init__.py`

Reply:
Done.